ramls/actionRequest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Action request",
+  "type": "object",
+  "properties": {
+    "amount": {
+      "type": "string",
+      "description": "Action amount"
+    },
+    "comments": {
+      "type": "string",
+      "description": "Additional info for staff or patron. Format: [STAFF : staff comment \n PATRON : patron comment]"
+    },
+    "transactionInfo": {
+      "type": "string",
+      "description": "Additional transaction information, e.g. check number"
+    },
+    "notifyPatron": {
+      "type": "boolean",
+      "description": "Should send patron notification or not"
+    },
+    "servicePointId": {
+      "type": "string",
+      "$ref": "uuid.json",
+      "description": "Unique ID of the service point where the payment was made"
+    },
+    "userName": {
+      "type": "string",
+      "description": "Name of the user that was logged in when the action was performed"
+    },
+    "paymentMethod": {
+      "type": "string",
+      "description": "Name of the payment method, e.g. `Cash` or `Credit card`"
```

Comment:
Does that make sense to define enum here?

Reply:
No, payment methods are custom (created by users).